testnets/xarchainceriestestnet/chain.json

```diff
@@ -0,0 +1,124 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "xarchainceriestestnet",
```

Comment:
Please exclude the testnet name "ceries" from the chain_name. All networks of the same project must use the same chain_name base.
(e.g., xarchainceries, xarchainceriestestnet)
vs
(e.g., xarchain, xarchaintestnet)

Reply:
ok. renamed it to "xarchaintestnet"